Source/Sentry/Private/Android/Infrastructure/SentryConvertorsAndroid.cpp

```diff
@@ -59,20 +59,11 @@ jobject SentryConvertorsAndroid::SentryMessageToNative(FString message)
 	jobject messageObject = Env->NewObject(messageClass, messageCtor);
 	jmethodID setMessageMethod = Env->GetMethodID(messageClass, "setMessage", "(Ljava/lang/String;)V");
 
-	Env->CallVoidMethod(messageObject, setMessageMethod, StringToNative(message));
+	Env->CallVoidMethod(messageObject, setMessageMethod, *FJavaClassObject::GetJString(message));
```

Comment:
Can you point me to API docs for `FJavaClassObject` - couldn't find it anywhere

Reply:
Apparently, there is no documentation for `FJavaClassObject`. You can probably check it out in UE sources ([AndroidJava.h](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Public/Android/AndroidJava.h), [AndroidJava.cpp](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Private/Android/AndroidJava.cpp)), though I'm not sure if that would be helpful.

Reply:
BTW why does the method arg need to be `(FString message)` instead of `(const FString& message)`?